filebeat/docs/inputs/input-docker.asciidoc

```diff
@@ -0,0 +1,75 @@
+:type: docker
+
+[id="{beatname_lc}-input-{type}"]
+=== Docker input
+
+++++
+<titleabbrev>Docker</titleabbrev>
+++++
+
+experimental[]
+
+Use the `docker` input to read logs from Docker containers. 
+
+This input searches for container logs under its path, and parse them into
+common message lines, extracting timestamps too. Everything happens before line
+filtering, multiline, and JSON decoding, so this input can be used in
+combination with those settings.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: docker
+  containers.ids: <1>
+    - '8b6fe7dc9e067b58476dc57d6986dd96d7100430c5de3b109a99cd56ac655347'
+----
+
+<1> `containers.ids` is required. All other settings are optional.
+
+==== Configuration options
+
+The `docker` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+[[config-container-ids]]
+===== `containers.ids`
+
+The list of Docker container IDs to read logs from. Specify
+`containers.ids: '*'` to read from all containers.
+
+===== `containers.paths`
+
+The base path where Docker logs are located. The default
+is `/var/lib/docker/containers`.
+
+// REVIEWERS: Not clear to me if this setting accepts an array or a string. If
+// it's just a string, why is it paths (plural)?
```

Comment:
@dedemorton I believe you are right here, it should be singular.

https://github.com/elastic/beats/blob/master/filebeat/input/docker/config.go#L15-L21

Nothing in the code currently accept multiple paths for the docker containers.

@exekias WDYT to move it to singular?

Reply:
Actually, the setting key is `path`, this looks like a typo in docs: https://github.com/elastic/beats/blob/master/filebeat/input/docker/config.go#L17

Reply:
Good catch @exekias